### ISSUE TITLE
Feature request: audio anti alias filter off

### ISSUE THREAD

**Issue by nsputnik** [fstark/macflim#44]
The audio quality is a little too good. Can there be a way to disable the anti alias filter on the encoding of both mp4 and flims so it sounds more like what we know 22khz 8 but audio to sound like? I have even tried encoding videos where I have used a bit reduction effect but once it is encoded it still sounds too good.

**Comment by fstark** [comment]
(note: I am partially deaf and unable to grasp most of the subtle sound encoding issues)

Let me rephrase and confirm that I understood you right.

_When encoding the 'mp4', the ffmpeg encoding machinery "corrects" the 22KHz sound, and makes it better than it should._

_However, when the flim file is played on original hardware, the sound is different._

_Using something like "aa=0" should disable the behavior._


The creation and configuration of the audio codec is at:

https://github.com/fstark/macflim/blob/main/src/writer.cpp#L268

This code was quite painful to write (and it shows :-))

If you have any idea what to change to get the desired effect, let me know. If not, I'll look at it later, if you confirm that I correctly understood the problem.

**Comment by fstark** [comment]
Re-reading your original comment, you seem to complain that the *flim* itself is also too good. Well, that 22KHz 8bits, it sounds exactly like it would have sound at the time, so I am not exactly sure what you’d like me to do there.
